README.md

```diff
@@ -48,16 +48,19 @@ To ease maintenance, the notebooks are made up of reusable components. To see wh
+[Cardinal setup](https://github.com/open-contracting/notebooks-ocds/blob/main/component_charts.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_charts.ipynb) | Install cardinal requirements, define coverage functions and calculate the field list for a given file_name.
+[Charts setup](https://github.com/open-contracting/notebooks-ocds/blob/main/component_setup_charts.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_setup_charts.ipynb) | Install charts requirements, import charts packages and define plot functions.
 [Kingfisher Process setup](https://github.com/open-contracting/notebooks-ocds/blob/main/component_setup_kingfisher.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_setup_kingfisher.ipynb) | Connect to the database. Choose the collection(s) and schema to work with.
 [Field list setup](https://github.com/open-contracting/notebooks-ocds/blob/main/component_setup_fieldlist.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_setup_fieldlist.ipynb) | Load the field list.
-[Data Registry setup](https://github.com/open-contracting/notebooks-ocds/blob/main/component_setup_registry.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_setup_registry.ipynb) | Install Cardinal, download the data, and calculate the field list.
+[Data Registry download data setup](https://github.com/open-contracting/notebooks-ocds/blob/main/ccomponent_setup_download_data_from_registry.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_setup_download_data_from_registry.ipynb) | Define the functions to list publications and download JSONL files from the registry.
+[Data Registry download data](https://github.com/open-contracting/notebooks-ocds/blob/main/component_select_data_from_registry.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_select_data_from_registry.ipynb) | Define the forms to select a publication and year and download the selected publication.
```

Comment:
These two components are only used together. We have a ton of components, so it'd be simpler to collapse them to one.

Reply:
Aha, but just for now, see my comment:

> Create one component to define the related data registry functions: list publications, get a select box for listing publications, get a select box to list the publication years, and download a publication given its ID and year. Right now this component is only used by component_select_data_from_registry, but it will also be used by the "check MVP criteria" notebook which will require downloading all the available publications, which is why it needs to be a component with functions that can be reusable for the mentioned use cases.

Reply:
👍